frontend/spec/features/spawning_spec.rb

```diff
@@ -6,6 +6,15 @@
   before(:all) do
     @repo = create(:repo, repo_code: "spawning_test_#{Time.now.to_i}")
     set_repo(@repo)
+    @accession = create(:json_accession,
+      title: "Spawned Accession",
+      extents: [build(:json_extent)],
+      dates: [build(:json_date)]
```

Comment:
I think `build(:json_date_single)` would work as well. Unfortunately, the `:json_date` factory uses a randomly selected `date_type` value.